### PR TITLE
Add release notes for gitops 1.2.1

### PR DESCRIPTION
**Pull request by reginaelyse**
These are the release notes for GitOps1.2.1, please review for accuracy @wtam2018 @iam-veeramalla@shubhamagarwal19 @jannfis, thanks! 

cc @Preeticp

### CODE REVIEW

File: modules/gitops-release-notes-1-2-1.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-2-1_{context}"]
+= Release notes for {gitops-title} 1.2.1
+
+{gitops-title} 1.2.1 is now available on {product-title} 4.7, 4.8, and 4.9.
```
Comment (jannfis) [inline modules/gitops-release-notes-1-2-1.adoc:8]: GitOps 1.2.x is not available for OCP 4.9

**Review by jannfis — CHANGES_REQUESTED** [review]
Looks good except one finding below.

Thanks @reginapizza!

File: modules/gitops-release-notes-1-2-1.adoc (hunk 1 above)
Comment (reginapizza) [reply modules/gitops-release-notes-1-2-1.adoc:8]: 1.2.1 is okay on 4.9 though right? I did include the note about 1.2 not be available on 4.9 under "known issues"

Comment (jannfis) [reply modules/gitops-release-notes-1-2-1.adoc:8]: No, the first release to support OCP 4.9 will be v1.3

File: modules/gitops-release-notes-1-2-1.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+this value can be overridden by specifying a number for `.spec.controller.kubeParallelismLimit` in the Argo CD CR spec.
+link:https://issues.redhat.com/browse/GITOPS-1025[GITOPS-1255]
+
+* The Kubernetes build failure caused by duplicate entries created in the kustomization.yaml when using the `kam bootstrap` command is now fixed. Pipeline has been updated to v0.24.2 and Tekton triggers have been updated to 0.14.2 to address this issue.
+link:https://issues.redhat.com/browse/GITOPS-1025[GITOPS-1273]
+
+* Previously persisting RBACs will now be automatically removed from target namespace when the Argo CD instance from source namespace is deleted.
+link:https://issues.redhat.com/browse/GITOPS-1025[GITOPS-1228]
+
+* Previously, when deploying an Argo CD instance into a namespace, the Argo CD instance would change the "managed-by" label  to be its own namespace. This fix would make namespaces unlabelled while also making sure the required RBACs are created and deleted for the namespace.
```
Comment (iam-veeramalla) [inline modules/gitops-release-notes-1-2-1.adoc:53]: I think there is an unintended spacing here -`"managed-by" label    to be its own namespace`

File: modules/gitops-release-notes-1-2-1.adoc [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+The following issues were resolved in the current release:
+
+* Previously, huge memory spikes were observed on the application controller on startup. The flag `--kubectl-parallelism-limit` for the application controller is now set to 10 by default, however
+this value can be overridden by specifying a number for `.spec.controller.kubeParallelismLimit` in the Argo CD CR spec.
+link:https://issues.redhat.com/browse/GITOPS-1025[GITOPS-1255]
+
+* The Kubernetes build failure caused by duplicate entries created in the kustomization.yaml when using the `kam bootstrap` command is now fixed. Pipeline has been updated to v0.24.2 and Tekton triggers have been updated to 0.14.2 to address this issue.
+link:https://issues.redhat.com/browse/GITOPS-1025[GITOPS-1273]
+
+* Previously persisting RBACs will now be automatically removed from target namespace when the Argo CD instance from source namespace is deleted.
```
Comment (iam-veeramalla) [inline modules/gitops-release-notes-1-2-1.adoc:50]: Does this sound better ?
`Remove RBACs from target-namespace when ArgoCD instance from source namespace is deleted`. 

p.s: I copied this from the JIRA description

File: modules/gitops-release-notes-1-2-1.adoc [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+[id="fixed-issues-1-2-1_{context}"]
+== Fixed issues
+The following issues were resolved in the current release:
+
+* Previously, huge memory spikes were observed on the application controller on startup. The flag `--kubectl-parallelism-limit` for the application controller is now set to 10 by default, however
+this value can be overridden by specifying a number for `.spec.controller.kubeParallelismLimit` in the Argo CD CR spec.
+link:https://issues.redhat.com/browse/GITOPS-1025[GITOPS-1255]
+
+* The Kubernetes build failure caused by duplicate entries created in the kustomization.yaml when using the `kam bootstrap` command is now fixed. Pipeline has been updated to v0.24.2 and Tekton triggers have been updated to 0.14.2 to address this issue.
+link:https://issues.redhat.com/browse/GITOPS-1025[GITOPS-1273]
```
Comment (iam-veeramalla) [inline modules/gitops-release-notes-1-2-1.adoc:48]: GITOPS-1025 ? Issues does not seem to be correct, Am I missing something ?

File: modules/gitops-release-notes-1-2-1.adoc [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+| TP
+|===
+
+[id="fixed-issues-1-2-1_{context}"]
+== Fixed issues
+The following issues were resolved in the current release:
+
+* Previously, huge memory spikes were observed on the application controller on startup. The flag `--kubectl-parallelism-limit` for the application controller is now set to 10 by default, however
+this value can be overridden by specifying a number for `.spec.controller.kubeParallelismLimit` in the Argo CD CR spec.
+link:https://issues.redhat.com/browse/GITOPS-1025[GITOPS-1255]
```
Comment (iam-veeramalla) [inline modules/gitops-release-notes-1-2-1.adoc:45]: GITOPS-1025 ? Issues does not seem to be correct, Am I missing something ?

File: modules/gitops-release-notes-1-2-1.adoc (hunk 4 above)
Comment (reginapizza) [reply modules/gitops-release-notes-1-2-1.adoc:48]: oh that is a mistake, I was using the link that was in the notes from last release so I got the format right and forgot to change both numbers, thanks for catching that. I'll fix it in all places

File: modules/gitops-release-notes-1-2-1.adoc (hunk 3 above)
Comment (reginapizza) [reply modules/gitops-release-notes-1-2-1.adoc:50]: I saw that and tried to write it in a way that sounded more like a full sentence... @Preeticp what do you think?

File: modules/gitops-release-notes-1-2-1.adoc (hunk 2 above)
Comment (reginapizza) [reply modules/gitops-release-notes-1-2-1.adoc]: I've removed that now, thanks for catching it!

File: modules/gitops-release-notes-1-2-1.adoc (hunk 1 above)
Comment (reginapizza) [reply modules/gitops-release-notes-1-2-1.adoc]: I've removed 4.9 now, thanks

**Comment by wtam2018** [comment]
@preethit PTAL

**Review by Preeticp — COMMENTED** [review]
@reginapizza I have made a few suggestions, PTAL

**Pull request merged**